### DISCUSSION TITLE
stream did not contain valid UTF-8

### DISCUSSION THREAD

**Discussion by thunder-flying** [maplibre/martin#1440] [Q&A]
The same error is reported when starting using cli or configuration file

martin -c world.mbtiles => Unable to load config file world.mbtiles: stream did not contain valid UTF-8

**Comment by sharkAndshark** [comment]
A config file should be there:
```bash
martin -c your_config_file.yml world.mbtiles
```
You could run Martin without config file either:
```bash
martin world.mbtiles
```

* [config doc](https://maplibre.org/martin/config-file.html)
* [CLI doc](https://maplibre.org/martin/run-with-cli.html)

**Comment by thunder-flying** [comment]
![image](https://github.com/user-attachments/assets/39bc22be-1249-4737-af86-2acc4230b9c7)
[config.yaml.zip](https://github.com/user-attachments/files/16372008/config.yaml.zip)

When I use the configuration file, the tile data access 404, as shown in the figure.
This is my config file.

**Comment by sharkAndshark** [comment]
I suppose you are using the `world_cities.mbtiles` from `Martin` repo?  
It's normal some requests is 404 because there is no data. But if every request is 404, there must be something wrong about the gridset.  
The gridset of this file is 3857. What's the gridset in your frontend code?
